.vscode/launch.json

```diff
@@ -0,0 +1,7 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": []
+}
```

Comment:
I am sure folder .vscode should be in .gitignore file. I think the same happens with .idea folder, but I am not sure in this case.

Reply:
sure